### PR TITLE
add spinnerview

### PR DESCRIPTION
**Pull request by noodlemoodle**
https://nanopay.atlassian.net/browse/NP-8350

### CODE REVIEW

**Review by lchanmann — APPROVED** [review]

File: src/foam/u2/view/SpinnerView.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'SpinnerView',
```
Comment (sarthak-marwaha) [inline src/foam/u2/view/SpinnerView.js:9]: How is this different from using `.tag(foam.u2.LoadingSpinner)`

Comment (noodlemoodle) [reply src/foam/u2/view/SpinnerView.js:9]: i need to push just a single spinner to the stack

Comment (sarthak-marwaha) [reply src/foam/u2/view/SpinnerView.js:9]: Does `ctrl.stack.push({class: 'foam.u2.LoadingSpinner', size: <some size>})` not work?

Comment (noodlemoodle) [reply src/foam/u2/view/SpinnerView.js:9]: ah i was using it as a view in a stackblock before so didnt work, what you suggested seems to work, thanks!

**Pull request closed without merging**